supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -29,51 +29,44 @@ object SharedFileSystem {
     }
   }
 
-  type PathsPair = (Path, Path)
-  type DuplicationStrategy = (Path, Path) => Try[Unit]
+  type FilesPair = (File, File)
+  type DuplicationStrategy = (File, File) => Try[Unit]
```

Comment:
I agree for the pair but for the strategy I think it's really easier to manipulate function `type`s than function `class`es